src/pwd/pwd.md

```diff
@@ -1,6 +1,6 @@
 # pwd
-pwd is a command used to get the current directory and print it in the terminal.
+Pwd is a command used to get the current directory and print it in the terminal.
```

Comment:
Since the command is in lowercase on the system, I suggest using backticks to highlight `pwd` differently (as in this comment), instead of capitalizing the first letter.

Reply:
Ok 👍

Reply:
I'll fix that shortly

Reply:
fixed